### DISCUSSION TITLE
Chatify for Laravel 9?

### DISCUSSION THREAD

**Discussion by roqstarco** [munafio/chatify#208] [General]
When will Chatify be available for Laravel 9?
Or what changes need to be done to use Chatify with Laravel 9?

Thanks.

**Comment by munafio** [comment]
> When will Chatify be available for Laravel 9?
> 
> Or what changes need to be done to use Chatify with Laravel 9?
> 
> 
> 
> Thanks.

It does support Laravel 9, just follow the documentation

**Reply by ghost** [reply]
I have it working on Laravel 9.

**Reply by roqstarco** [reply]
Yes, got it working too but had to do some tweaks.
